### ISSUE TITLE
keep image ratio in ppt when  converting document

### ISSUE THREAD

**Issue by xiaozhubenben** [ONLYOFFICE/DocumentServer#391]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
I use document conversion service to convert a ppt to png images.
After converting,some pages seemed converted right, but  some   images embeded in  the ppt have  been scaled incorrectly,

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
send  the attached Powerpoint file to Document Server Conversion Service with these params
```
 var params = {
        async: false,
        url: documentUri,
        outputtype: 'zip',
        "thumbnail": {
            "aspect": 1,
            "first": false,
            "height": 1000,
            "width": 1000
        },
        filetype: fromExtension.replace(".", ""),
        title: title,
        key: documentId
    };
```

**What is the expected behavior?**
Keep image correct ratio after converting.

**Did this work in previous versions of DocumentServer?**
n/a
**DocumentServer version:**
5.1.5
**Operating System:**
DS installed on ubuntu 16.04 64bits

**Browser version:**
n/a

origin file:
[file.pptx](https://github.com/ONLYOFFICE/DocumentServer/files/2374439/file.pptx)

one of incorrect  image:
![error](https://user-images.githubusercontent.com/12681822/45415590-9380ef00-b6b0-11e8-8776-c30ca854b079.png)

correct image converted use powerpoint:
![correct](https://user-images.githubusercontent.com/12681822/45415582-8e23a480-b6b0-11e8-8d2d-5bba669d68bc.png)

**Comment by ShockwaveNN** [comment]
Thank you. I can confirm - this is a bug
I created an issue 38900 in our private bug tracker

**Comment by jiriks74** [comment]
@ShockwaveNN From my testing, this bug is resolved

**Comment by ShockwaveNN** [comment]
@jiriks74 I'll agree, thanks, seems we forgot about it since 2018

This issue should be fixed in v7.1.0 (I've rechecked it) so I'll close it, feel free to comment
